package.json

```diff
@@ -77,5 +77,9 @@
     "transformIgnorePatterns": [
       "node_modules/(?!(jest-)?react-native|react-clone-referenced-element|react-navigation-deprecated-tab-navigator|react-navigation)"
     ]
+  },
+  "prettier": {
```

Comment:
This is the same config as the eslint plugin, prettier has a cool setting where it only formats on save when it finds a config in the project.